system_stats.h

```diff
@@ -59,7 +59,7 @@ typedef struct {
 	float avg10;
 	float avg60;
 	float avg300;
-	float total;
+	uint64 total;
```

Comment:
fscanf also should be changed: https://github.com/CyberDem0n/bg_mon/blob/3a64b237d8828eda5782627e7ddc33fa744bd2d9/system_stats.c#L103